File: JS/car.js
function Car(license, driver) {
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger;
}

Car.prototype.printDataCar = function () {
    console.log(this.driver)
    console.groupCollapsed(this.driver.name)
    console.log(this.driver.document)
}

File: java/Car.java
class Car {
    Integer id;
    String license;
    String driver;
    Integer passegenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("license: " + license +  " Driver: " + driver);
    }
}